src/sentry/api/endpoints/organization_events.py

```diff
@@ -126,3 +126,44 @@ def data_fn(offset, limit):
                         request, organization, params["project_id"], results
                     ),
                 )
+
+
+class OrganizationEventsGeoEndpoint(OrganizationEventsV2Endpoint):
+    def get(self, request, organization):
+        if not self.has_feature(organization, request):
+            return Response(status=404)
+
+        try:
+            params = self.get_snuba_params(request, organization)
+        except NoProjects:
+            return Response([])
+
+        fields = request.GET.getlist("field")
+        fields = ["geo.country_code", fields[0]]
```

Comment:
Should this just be
```suggestion
        fields = ["geo.country_code", request.GET.get("field")]
```
and add some basic validation on `field`.